package.json

```diff
@@ -90,15 +90,14 @@
     "babel-preset-es2015": "^6.18.0",
     "babel-preset-react": "^6.16.0",
     "chalk": "^2.4.1",
-    "color": "^3.0.0",
     "colors": "^1.3.2",
-    "copy-webpack-plugin": "^4.5.2",
+    "copy-webpack-plugin": "^4.5.3",
```

Comment:
I checked this locally - the readme for the docs does indeed still get copied over 😅